products/sle/main.pm

```diff
@@ -990,31 +990,63 @@ sub load_slenkins_tests {
 
 sub load_ha_cluster_tests {
     return unless (get_var("HA_CLUSTER"));
-    loadtest "ha/wait_support_server";
+    loadtest "installation/grub_test";
```

Comment:
Wouldn't be better to use boot/boot_to_desktop instead of combination grub_test/first_boot?

Reply:
Yes you're right, it's better.
Next version will use boot/boot_to_desktop instead!

Reply:
I think that the same think should be also changed for symlinks related to fencing_*. There should be only one pointing to boot_to_desktop.

Reply:
Yes, it was done too for my next version.